index.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "client-widget-CREA-fcpo-specials-banner",
   "beta": true,
-
+  "is_layout": true,
+  "parent_component_source_uri": "git@github.com:g5components/cms-widget-featured-content-pop-out.git",
```

Comment:
We have some "hardcoded" CMS concerns https://github.com/g5search/g5-content-management-system/blob/master/app/models/concerns/widgets/featured_content_pop_out_widget.rb

When `parent_component_source_uri` is set on a widget the component will "act like" the parent.

I'm of the opinion that all configuration should come from the index.json and not have any hard-coded stuff in the CMS